src/amo/components/Addon/index.js

```diff
@@ -385,6 +386,22 @@ export class AddonBase extends React.Component {
     );
   }
 
+  renderContributeCard() {
+    const { addon } = this.props;
+
+    if (!addon || (addon && !addon.contributions_url)) {
+      return null;
+    }
+
+    return (
+      <Contribute
+        addonType={addon.type}
+        authorNames={addon.authors.length}
```

Comment:
This prop should be `numberOfAuthors`. Even though `Addon/index.js` doesn't use Flow, we still need to address https://github.com/mozilla/addons-frontend/issues/3563 before this type of thing can be caught.

Reply:
oh no, I badly rebased it seems 😕